src/Dfe.EarlyYearsQualification.Content/Entities/PhaseBanner.cs

```diff
@@ -0,0 +1,14 @@
+using Contentful.Core.Models;
+
+namespace Dfe.EarlyYearsQualification.Content.Entities;
+
+public class PhaseBanner
+{
+  public string PhaseName { get; init; } = string.Empty;
+
+  public Document? Content { get; set; }
+
+  public string ContentHtml {get; set; } = string.Empty;
```

Comment:
question: should this be `init`?

We don't want code to be able to change this? The property would then have to be either nullable `string?`, or settable by constructor parameter.